Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -181,6 +181,54 @@ lemma c_relation : 1728 * W.Δ = W.c₄ ^ 3 - W.c₆ ^ 2 := by
 
 end Quantity
 
+section ModelsWithJInvariant_0_1728
+
+variable (R : Type u) [CommRing R]
+
+/-- The Weierstrass curve $Y^2 + Y = X^3$. -/
+def ofJInvariant0 : WeierstrassCurve R :=
```

Comment:
The j-invariant in this file is called `j` rather than `jInvariant`, but I think `jInvariant` is also good as long as we're consistent. I guess you should either change this to `J`, or rename all other existing instances of `j` into `jInvariant` (or even something funny like `\MiJ` for italics). What do you think?

Reply:
I think it's good. Later I'll try to rename all `onJInvariant` to `ofJ` and see if it's better.